onmt/translate/decode_strategy.py

```diff
@@ -1,4 +1,38 @@
 import torch
+import onmt
+
+
+def get_decode_strategy(beam_size, batch_size, bos, eos, pad, device,
+                        min_length, max_length, block_ngram_repeat,
+                        exclusion_tokens, return_attention, memory_lengths,
+                        keep_topk=1, sampling_temp=1,  # random_sampling only
+                        ratio=0., stepwise_penalty=None,  # beam search only
+                        n_best=1, global_scorer=None):
+    if beam_size == 1:
+        strategy = onmt.translate.RandomSampling(
```

Comment:
It seems clearer to name this `GreedySearch` as this is what it does by default when `beam_size == 1`.